external/webserver/http_client.h

```diff
@@ -50,6 +50,8 @@ struct http_client_t {
 	mbedtls_ssl_context       tls_ssl;
 	mbedtls_net_context       tls_client_fd;
 #endif
+
+	int keep_alive;
```

Comment:
As checked, this variables set only 0 or 1. 
How about using bool type instead of int?
It is much better to read and can save the memory.